app/worker.ts

```diff
@@ -262,3 +307,7 @@ self.addEventListener("message", async (event: { data: any }) => {
     data: "OK",
   });
 });
+
+(async () => {
+  aiMaskClient = await AIMaskClient.getWorkerClient()
```

Comment:
Why not just `await` above as well as needed? It doesn't seem like it should be a long running operation?

This seems like it'll cause a race condition?

Reply:
That is supposed to be executed be executed at startup. I added an await on the main thread to verify this is done before doing anything else. 

Agree that events in the worker are not safe guarded, could be better implemented but I think it's enough for here.